manifests/valve.pp

```diff
@@ -189,11 +191,26 @@
 
   # only run if ensure is not absent
   if ! ($ensure == absent) {
-    # run augeas with our changes
-    augeas {
-      'repose_sysconfig':
-        context => '/files/etc/sysconfig/repose',
-        changes => [ $repose_sysconfig, $saxon_sysconfig ]
+    if $::systemd == 'true' {
+      file { "/etc/systemd/system/repose-valve.service.d":
+        ensure => 'directory',
+        owner  => 'root',
+        group  => 'root',
+        mode   => '0755',
+      }
+        
+      file {"/etc/systemd/system/repose-valve.service.d/local.conf":
```

Comment:
I wasn't thrilled with that implementation. Using file resources to get what I want may be inelegant, but it's very readable and you know what you are ending up with.